### PR TITLE
Background

### PR DESCRIPTION
**Pull request by mehmetozer98**
rettet ift. review

### CODE REVIEW

File: foundation/section/Background.tex [hunk 1]
```diff
@@ -0,0 +1,6 @@
+202,2 million barrels of beer were shipped in 2018 by U.S. beer industry, which lead us to the conclusion
+that beer is often drunk and is a popular drink on  the table. This motivates the beer companies to brew faster
+and more automated than they did before.\footnote{\url{https://www.nbwa.org/resources/industry-fast-facts}}
+The overall purpose is to make the manual processes automatic by having the brewing machine communicate with other interfaces.
+In this way, they can increase the speed of each batch by estimating the error functions and using the optimal
```
Comment (mdhenriksen) [inline foundation/section/Background.tex]: How does estimating error functions make them able to increase speed?

File: foundation/section/Background.tex [hunk 2]
```diff
@@ -0,0 +1,6 @@
+202,2 million barrels of beer were shipped in 2018 by U.S. beer industry, which lead us to the conclusion
```
Comment (Kargo) [inline foundation/section/Background.tex]: > 202,2 million barrels of beer were shipped in 2018 by U.S. beer industry, which lead**s** us to the conclusion

Manglende s

File: foundation/section/Background.tex [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+that beer is often drunk and is a popular drink on  the table. This motivates the beer companies to brew faster
```
Comment (Kargo) [inline foundation/section/Background.tex]: > that beer is often drunk and is a popular drink on  the table.

Ville nok bytte rundt på lidt ord. Det får sætningen til at flyde mere naturligt.
> that beer is a popular drink that is often seen on the table.

File: foundation/section/Background.tex (hunk 1 above)
Comment (mehmetozer98) [reply foundation/section/Background.tex]: rettet

File: foundation/section/Background.tex (hunk 2 above)
Comment (mehmetozer98) [reply foundation/section/Background.tex]: ok

File: foundation/section/Background.tex (hunk 3 above)
Comment (mehmetozer98) [reply foundation/section/Background.tex]: set

File: foundation/section/Background.tex [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+and more automated than they did before.\footnote{\url{https://www.nbwa.org/resources/industry-fast-facts}}
+The overall purpose is to make the manual processes automatic by having the brewing machine communicate with other interfaces.
```
[the remark this answers is not in the document]
Comment (mehmetozer98) [reply foundation/section/Background.tex]: rettet

**Pull request merged**